7_deploy_osh/roles/suse-osh-deploy/templates/cinder.yaml.j2

```diff
@@ -1,19 +1,19 @@
 conf:
   ceph:
     enabled: true
-    admin_keyring: {{ ceph_admin_keyring_b64key | b64decode }}
+    admin_keyring: {{ ceph_admin_keyring_key }}
   cinder:
     DEFAULT:
       debug: true
       backup_driver: cinder.backup.drivers.ceph
-      backup_ceph_user: {{ ses_cluster_configuration['cinder-backup']['rbd_store_user'] }}
+      backup_ceph_user: {{ ses_cluster_configuration['cinder-backup']['rbd_store_pool_user'] }}
```

Comment:
This change should be reflected in the example `ses_config.yml ` (not just cinder)